starlite/datastructures/provide.py

```diff
@@ -15,30 +13,24 @@
 class Provide:
     """A wrapper class for dependency injection."""
 
-    __slots__ = ("dependency", "use_cache", "cache_per_request", "cache_key", "lock", "value", "signature_model")
+    __slots__ = ("dependency", "use_cache", "value", "signature_model")
 
     def __init__(
         self,
         dependency: "AnyCallable",
         use_cache: bool = False,
-        cache_per_request: bool = False,
-        cache_key: Optional[str] = None,
         sync_to_thread: bool = False,
+        **_kwargs: Any,
```

Comment:
what are kwargs used for?

Reply:
Thought this might be better than just removing the `cache_per_request` and `cache_key` options. I can remove this if that's not a concern.